__init__.py

```diff
@@ -0,0 +1,149 @@
+# Regina Bloomstine, Casimiro Ferreira, Andrii Pernatii, Kirill Hrymailo
+# BSD-3 License
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+# 1. Redistributions of source code must retain the above copyright notice,
+#    this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+# 3. Neither the name of the copyright holder nor the names of its
+#    contributors may be used to endorse or promote products derived from this
+#    software without specific prior written permission.
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+# CONTRIBUTORS  BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA,
+# OR PROFITS;  OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE,  EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+from neon_utils.skills.neon_skill import NeonSkill, LOG
+from mycroft.skills.core import intent_file_handler
+from .request_handling import RequestHandler
+
+import os
+
+
+class MallParserSkill(NeonSkill):
+
+    def __init__(self):
+
+        super(MallParserSkill, self).__init__(name="MallParserSkill")
+        self.script_path = os.path.join(os.path.dirname(__file__),
+                                        'mall')
+        self.request_handler = object
+
+    def initialize(self):
+        # When first run or prompt not dismissed, wait for load and prompt user
+        if self.settings.get('prompt_on_start'):
+            self.bus.once('mycroft.ready', self._start_mall_parser_prompt)
+
+    @intent_file_handler("run_mall_parser.intent")
+    def start_mall_parser_intent(self, message):
+        LOG.info(message.data)
+
+        self._start_mall_parser_prompt(message)
+        return
+
+    def finish(self):
+        self.speak_dialog("finished")
+
+    def repeat(self):
+        self.speak_dialog("repeat")
+
+    def no_lang(self):
+        self.speak_dialog("no_lang")
+
+    def what_shop(self):
+        user_request = self.get_response("what_shop")
+        return user_request
+
+    def select(self, variants):
+        self.speak_dialog('more_than_one')
+        choice = self.get_response(variants)
+        return choice
```

Comment:
There's actually already a method for asking the user to select an option from a list: https://github.com/OpenVoiceOS/ovos-core/blob/55c8d08a2a9201556af98d21d5b1b0ef6181a030/mycroft/skills/mycroft_skill/mycroft_skill.py#L855